tests/test_commands.py

```diff
@@ -68,3 +70,150 @@ def _test_command(arg):
     assert len(basicApp.commands.get()) == 1
     assert len(plugin.commands.get()) == 1
     assert len(plugin2.commands.get()) == 0
+
+
+def test_command_mandatory_argument(basicApp):
+    def _test_command(*args, **kwargs):
+        print(args)
+        print(kwargs)
+
+    plugin = basicApp.plugins.get("CommandPlugin")
+    plugin.commands.unregister("test")
+    # register a command with a mandatory argument
+    plugin.commands.register("test", "my test command", _test_command, params=[Argument(("man_arg",), required=True)])
+    # call command with argument -> expect ok
+    result = CliRunner().invoke(basicApp.commands.get("test").click_command, ["arg"])
+    assert result.exit_code == 0
```

Comment:
I tried to capture the stdout, stderr mit capsys, but it shows nothing. I would say it is enough though, because in the TC I do two calls, the only difference being the two, is that the argument is missing in the one.

Reply:
I'm ok with that reasoning. I'm having trouble with capsys in the past too.